lib/src/sql/escape.rs

```diff
@@ -15,31 +16,36 @@ pub fn escape_strand(s: &str) -> String {
 }
 
 #[inline]
-pub fn escape_id(s: &str) -> String {
-	for x in s.chars() {
-		if !val_char(x) {
-			return format!("{}{}{}", BRACKET_L, s, BRACKET_R);
+pub fn escape_id(s: &str) -> Cow<'_, str> {
+	for x in s.bytes() {
+		if !val_u8(x) {
+			return Cow::Owned(format!("{}{}{}", BRACKET_L, s, BRACKET_R));
```

Comment:
As `val_*()` only cares about ASCII, it is correct (and much faster) to iterate `bytes()` instead of `chars()`.